velox/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ if(${VELOX_ENABLE_PARSE})
 endif()
 
 # hive connector depends on dwio
-if(${VELOX_ENABLE_HIVE_CONNECTOR})
+if(${VELOX_ENABLE_HIVE_CONNECTOR} OR ${VELOX_BUILD_PYTHON_PACKAGE})
```

Comment:
PyVelox doesnt depend on dwio right ?

Reply:
It does but it wasn't properly linked before so it was not throwing an error. I guess it was using header only things? It's transitively via common/compression as that includes dwio/common/exception/Exceptions.h

We can remove this if we(you^^) can confirm that it's header only though we really should create header only targets for such cases to avoid this implicit connections... or leave a comment for now

Reply:
I cant see why pyVelox should depend on anyway on dwio - not even header only , could be a case of the dependency coming in transitively (or via test), but then I could be very wrong . Can you paste the error you see if you try and build pyvelox without dwio ?